spec/lib/stripe/credit_card_cloner_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'stripe/credit_card_cloner'
+
+module Stripe
+  describe CreditCardCloner do
+    describe "#clone!" do
+      let(:cloner) { Stripe::CreditCardCloner.new }
+
+      let(:customer_id) { "cus_A123" }
+      let(:card_id) { "card_1234" }
+      let(:new_customer_id) { "cus_A456" }
+      let(:new_payment_method_id) { "pm_4567" }
+      let(:stripe_account_id) { "acct_456" }
+      let(:customer_response_mock) { { status: 200, body: customer_response_body } }
+
+      let(:credit_card) { create(:credit_card, user: create(:user)) }
+
+      before do
+        allow(Stripe).to receive(:api_key) { "sk_test_12345" }
+
+        stub_request(:post, "https://api.stripe.com/v1/customers")
```

Comment:
I'm wondering if it's time for the vcr gem. We used it for Fair Food and it's working really well. It records real API responses to repeat them offline. So you get real data and you are setup to repeat it against the live API whenever you want.

Reply:
I like this webmock way of testing things but @kristinalim will be looking into testing these things, maybe a good pointer for #4639